internal/pkg/mapper/defaultbucket/local_save.go

```diff
@@ -0,0 +1,72 @@
+package defaultbucket
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
+)
+
+// MapBeforeLocalSave - replace default buckets in IM with placeholders.
+func (m *defaultBucketMapper) MapBeforeLocalSave(recipe *model.LocalSaveRecipe) error {
+	config, ok := recipe.Object.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	inputTablesRaw := utils.GetFromMap(config.Content, []string{"storage", "input", "tables"})
+	inputTables, ok := inputTablesRaw.([]interface{})
+	if !ok {
+		return nil
+	}
+
+	for _, inputTableRaw := range inputTables {
+		inputTable, ok := inputTableRaw.(*orderedmap.OrderedMap)
+		if !ok {
+			continue
+		}
+		inputTableSourceRaw, ok := inputTable.Get(`source`)
+		if !ok {
+			continue
+		}
+		inputTableSource, ok := inputTableSourceRaw.(string)
+		if !ok {
+			continue
+		}
```

Comment:
Tahle část je společná pro load i save, ale nejsem si jistý jestli stojí za to to obstrahovat do nějaké společné metody. Co myslíš?

Reply:
asi ma, je to dost dlhe ... mozno nieco ako `visitTables(callback)`?

Reply:
ok 👍

Reply:
Tak hotovo.